Protocol.md

```diff
@@ -627,6 +631,13 @@ subprotocol strings, and:
   SHALL validate that the negotiated subprotocol is present in the
   *subprotocols* field.
 
+Furthermore, the server SHALL validate that the *ping_interval* field
+contains a non-negative integer. If the value is `0`, the server SHALL
+NOT send WebSocket *ping* messages to the client. Otherwise, the server
```

Comment:
That would open up the possibility to send *ping* messages regardless of what the client requested. That's unwanted (so far).